hotelservice/Services/Hotel/HotelService.cs

```diff
@@ -207,16 +237,16 @@ public async Task<HotelCancelBookRoomsResponse> CancelBookRooms(HotelCancelBookR
 
     public async Task<GetPopularHotelsResponse> GetPopularHotels(GetPopularHotelsRequest request)
```

Comment:
I think it would be a good idea to change the return type from tuple to Dict, since tuples are not displayed correctly in  API gateway's response
```
"Name": hotel.Name,
"Country": hotel.Country,
"City": hotel.City
```

Reply:
Fixed.